app/assets/stylesheets/application/styles/_event_show.scss

```diff
@@ -0,0 +1,262 @@
+    float: left;
+    opacity: 1;
+    width: 33.33%;
+  }
+}
+
+.event-show__large-section {
+  background-color: $pale-gray-color;
+  text-align: center;
+  padding: 16px;
+}
+
+.event-show__title {
+  @extend %heading-3;
+}
+
+.event-show__location {
+  margin-top: 12px;
+}
+
+.event-show__event-date {
+  margin-top: 6px;
+}
+
+.event-show__event-time {
+  margin-top: 6px;
+}
+
+.event-show__event-price {
+  margin-top: 6px;
+}
+
+
+.event-show__map-link {
+  text-decoration: underline;
+
+  &:hover {
+    color: $primary-color;
+  }
+}
+
+.event-show__host {
+  @include span(12 of 12);
+  padding-bottom: 16px;
+  padding-top: 16px;
+
+  @include breakpoint($tablet) {
+    @include span(5 of 12);
+    box-sizing: border-box;
+    padding-right: gutter(16px);
+  }
+
+  @include breakpoint($desktop) {
+    @include span(4 of 12);
+  }
+}
+
+.event-show__host-title {
+  @extend %heading-4;
+  margin: 8px 0 16px;
+  text-align: center;
+}
+
+.event-show__host-avatar {
+  border: 3px solid $gray-color;
+  border-radius: 50%;
+  display: block;
+  height: 200px;
+  margin: 0 auto;
+  object-fit: cover;
+  width: 200px;
+}
+
+.event-show__host-full-name {
+  @extend %heading-6;
+
+  margin-top: 24px;
+  text-align: center;
+}
+
+.event-show__host-bio {
+  margin: 16px auto 0;
+  max-width: $desktop;
+  padding: 0 16px;
+  text-align: center;
+}
+
+.event-show__event-details {
+  @include span(12 of 12);
+
+  @include breakpoint($tablet) {
+    @include span(last 7 of 12);
+  }
+
+  @include breakpoint($desktop) {
+    @include span(last 8 of 12);
+  }
+}
+
+.event-show__event-description {
+  background-color: $pale-gray-color;
+  padding: 16px;
+
+  @include breakpoint($tablet) {
+    border-bottom: 1px solid $light-gray-color;
+    background-color: transparent;
+  }
+}
+
+.event-show__event-menu {
+  padding: 16px;
+}
+
+.event-show__event-detail-title {
+  @extend %heading-4;
+  padding-bottom: 8px;
+}
+
+.event-show__markdown {
+  h1 {
+    @extend %heading-5;
+    padding-bottom: 8px;
+    padding-top: 8px;
+  }
+
+  h2, h3, h4, h5, h6 {
+    @extend %heading-6;
+    padding-bottom: 8px;
+    padding-top: 8px;
+  }
+
+  p {
+    padding-bottom: 8px;
+  }
+
+  blockquote {
+    border-left: 1px solid $primary-color;
+    background-color: lighten($pale-gray-color, 2%);
+    padding: 16px 16px 8px;
+  }
+
+  hr {
+    margin-top: 22px;
+    padding-top: 22px;
+  }
+
+  ul, ol {
+    padding-bottom: 8px;
+  }
+}
+
+.event-show__event-booking {
+  @include span(12 of 12);
+  background-color: $pale-gray-color;
+  padding-bottom: 16px;
+  padding-top: 16px;
+  text-align: center;
+
+  .stripe-button-el {
+    background: none;
+    border-radius: 0;
+    box-shadow: none;
+    @extend .event-show__booking-button;
+    padding: 0;
+
+    span {
+      @extend %secondary-font;
+      background: none;;
+      border-radius: 0;
+      box-shadow: none;
+      font-size: 1em;
+      font-weight: normal;
+      text-shadow: none;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, font-size, font-weight, padding, text-shadow, word-spacing